app/src/main/java/grinnell/appdev/edu/lyles/AsyncRetrieval.java

```diff
@@ -0,0 +1,69 @@
+package grinnell.appdev.edu.lyles;
+
+import android.os.AsyncTask;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+/**
+ * Created by Shelby on 2/3/2017.
```

Comment:
I like that you started a Javadoc but it'll be beneficial if you added more information about what this class is/what it does. Read on Javadoc and let's update this documentation : https://www.tutorialspoint.com/java/java_documentation.htm